source/db/protocols/eo-protocols.json

```diff
@@ -18,6 +18,15 @@
     }]
   },
   {
+    "name": "Matrix",
```

Comment:
This didn't get addressed; please remove it. It'll be inherited from the English version, remember.